src/base/bittorrent/torrentimpl.cpp

```diff
@@ -1864,37 +1871,23 @@ void TorrentImpl::handleFileRenamedAlert(const lt::file_renamed_alert *p)
     // Remove empty leftover folders
     // For example renaming "a/b/c" to "d/b/c", then folders "a/b" and "a" will
     // be removed if they are empty
-    const QString oldFilePath = m_filePaths.at(fileIndex);
-    const QString newFilePath = Utils::Fs::toUniformPath(p->new_name());
+    const Path oldFilePath = m_filePaths.at(fileIndex);
+    const auto newFilePath = Path(QString(p->new_name()));
```

Comment:
declaration style
seems you never review it again your selves after you code it...

Reply:
>seems you never review it again your selves after you code it...

It was redesigned several times even before I published PR... It was something like `const auto newFilePath = Path::fromString(QString(p->new_name()));` previously.